spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -292,7 +294,20 @@ public SparkContext createSparkContext() {
     }
     if (System.getenv("SPARK_HOME") != null) {
       conf.setSparkHome(System.getenv("SPARK_HOME"));
+
+      // load configs from spark properties file
+      final String sparkPropertiesFile =
+          Utils.getDefaultPropertiesFile(JavaConversions.asScalaMap(System.getenv()));
```

Comment:
I think SPARK_HOME might not be set (if using the included Spark build in Zeppelin?) and Utils.getDefaultPropertiesFile() doesn't seem to handle it in that case?